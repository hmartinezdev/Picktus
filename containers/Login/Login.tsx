import React, { Component } from 'react';
import Logo from '@components/Logo';
import Background from '@components/Background';

class Login extends Component {
  render() {
    return (
      <Background>
        <div className="container">
          <div className="login">
            <Logo />
          </div>
          <style jsx>{`
            .container {
              width: 100%;
              height: 100vh;
              display: flex;
              justify-content: center;
              align-items: flex-start;
              padding: 2rem;
              box-sizing: border-box;
            }

            .login {
              background-color: #2d3047cd;
              padding-right: 2rem;
              box-shadow: 11px 10px 34px -7px rgba(0, 0, 0, 0.49);
            }
          `}</style>
        </div>
      </Background>
    );
  }
}

export default Login;
