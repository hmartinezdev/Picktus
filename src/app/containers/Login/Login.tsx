import React, { Component } from 'react';
import Logo from '@components/Logo';
import Background from '@components/Background';
import colors from '@constants/colors';

class Login extends Component {
  render() {
    return (
      <Background>
        <div className="container">
          <div className="login">
            <Logo />
          </div>
          <style jsx>{`
            .container {
              width: 100%;
              height: 100vh;
              display: flex;
              justify-content: center;
              align-items: flex-start;
              padding: 2rem;
              box-sizing: border-box;
            }

            .login {
              background-color: ${colors.primary}cd;
              padding-right: 2rem;
              box-shadow: 11px 10px 34px -7px rgba(0, 0, 0, 0.49);
              display: flex;
              position: relative;
            }

            .login::after {
              position: absolute;
              top: 50%;
              left: 50%;
              transform: translate(-50%, -50%);
              content: '';
              width: calc(100% + 2rem);
              height: calc(100% + 2rem);
              border: 3px solid transparent;
              border-image: linear-gradient(to bottom, ${colors.primary} 60%, ${colors.secondary} 100%);
              border-image-slice: 1;
            }
          `}</style>
        </div>
      </Background>
    );
  }
}

export default Login;
