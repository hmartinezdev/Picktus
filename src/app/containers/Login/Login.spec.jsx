import { shallow } from 'enzyme';
import React from 'react';
import Login from './Login';

let wrapper;

const setup = () =>
  shallow(
    <Login>
      <div />
    </Login>
  );

describe('<Login />', () => {
  test('should render properly', () => {
    wrapper = setup();
    expect(wrapper).toMatchSnapshot();
  });
});
