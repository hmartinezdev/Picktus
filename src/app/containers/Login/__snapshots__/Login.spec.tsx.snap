// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Login /> should render properly 1`] = `
<Background>
  <div
    className="container"
  >
    <div
      className="login"
    >
      <Logo />
    </div>
    <style
      jsx={true}
    >
      
            .container {
              width: 100%;
              height: 100vh;
              display: flex;
              justify-content: center;
              align-items: flex-start;
              padding: 2rem;
              box-sizing: border-box;
            }

            .login {
              background-color: #2d3047cd;
              padding-right: 2rem;
              box-shadow: 11px 10px 34px -7px rgba(0, 0, 0, 0.49);
              display: flex;
              position: relative;
            }

            .login::after {
              position: absolute;
              top: 50%;
              left: 50%;
              transform: translate(-50%, -50%);
              content: '';
              width: calc(100% + 2rem);
              height: calc(100% + 2rem);
              border: 3px solid transparent;
              border-image: linear-gradient(to bottom, #2d3047 60%, #FAC9B8 100%);
              border-image-slice: 1;
            }
          
    </style>
  </div>
</Background>
`;
